values.yaml

```diff
@@ -9,23 +9,31 @@ georchestra:
     # If images are publicly available, the parameter should be left commented
     # out.
     #
+    # Set replicas to "0" to turn off an app for maintenance as discussed
+    # in https://github.com/georchestra/helm-georchestra/issues/42.
+    # For most of the apps, replicas are not expected to be above 1
```

Comment:
```suggestion
    # For all the apps, setting a replica count of more than 1 is not supported by the helm chart.
```

Reply:
Until this has been tested, I would prefer to say that it's not supported.

Reply:
Right. Done